atom/common/api/atom_api_command_line.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "base/command_line.h"
+#include "native_mate/converter.h"
+#include "native_mate/dictionary.h"
+
+#include "atom/common/node_includes.h"
+
+namespace {
+
+bool HasSwitch(const base::CommandLine::StringType& name) {
+  return base::CommandLine::ForCurrentProcess()->HasSwitch(name);
+}
+
+base::CommandLine::StringType GetSwitchValue(
```

Comment:
`#include "atom/common/native_mate_converters/string16_converter.h"` for `StringType` on windows.